Formula/pixie.rb

```diff
@@ -0,0 +1,31 @@
+class Pixie < Formula
+  desc "Observability tool for Kubernetes applications"
+  homepage "https://px.dev/"
+  url "https://github.com/pixie-io/pixie.git",
+      tag:      "release/cli/v0.7.14",
+      revision: "c15a89eebe0849f16b83ac369cd21d2e6911b1a2"
+  license "Apache-2.0"
+  head "https://github.com/pixie-io/pixie.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    semver = build.head? ? "0.0.0-dev" : version
+    ldflags = %W[
+      -X px.dev/pixie/src/shared/goversion.buildSCMRevision=#{Utils.git_short_head}
+      -X px.dev/pixie/src/shared/goversion.buildSCMStatus=Homebrew
+      -X px.dev/pixie/src/shared/goversion.buildSemver=#{semver}
+      -X px.dev/pixie/src/shared/goversion.buildTimeStamp=#{time.to_i}
+      -X px.dev/pixie/src/shared/goversion.buildNumber=#{revision + (bottle.nil? ? 1 : bottle.rebuild + 1)}
+    ]
+    system "go", "build", *std_go_args(ldflags: ldflags, output: bin/"px"), "./src/pixie_cli/px.go"
+  end
+
+  test do
+    stdin, stdout, stderr, = Open3.popen3("#{bin}/px version")
+    stdin.close
+    assert_match "Pixie CLI", stderr.read
+    stdout.close
+    stderr.close
```

Comment:
Why not just `shell_output` with `2>&1`?

Though, we need a proper functional test rather than one that just calls `version`.

Reply:
We can run `version` or `--help`. 
All other commands require you to have authenticated with pixie and have a Kubernetes cluster (to deploy pixie to, or to query/get data from).